src/sync/installation.test.ts

```diff
@@ -229,7 +229,6 @@ describe("sync/installation", () => {
 			githubUserTokenNock(DatabaseStateCreator.GITHUB_INSTALLATION_ID);
 			githubNock
 				.post("/graphql")
-				.times(2)
```

Comment:
Was failing because we were retrying it even though it wasn't changedFiles error 🤦